packages/editor-ui/.eslintrc.js

```diff
@@ -17,7 +17,6 @@ module.exports = {
 		'import/order': 'off',
 		'import/no-cycle': 'warn',
 		'import/no-duplicates': 'warn',
-		indent: 'warn',
```

Comment:
this was unnecessary, as we disable this rule globally, so that prettier can take care of all code indentation.